src/components/ha-picture-upload.ts

```diff
@@ -0,0 +1,193 @@
+import {
+  LitElement,
+  TemplateResult,
+  html,
+  property,
+  css,
+  customElement,
+  internalProperty,
+  PropertyValues,
+} from "lit-element";
+import { createImage, generateImageThumbnailUrl } from "../data/image";
+import { HomeAssistant } from "../types";
+import { fireEvent } from "../common/dom/fire_event";
+import "./ha-circular-progress";
+import { mdiClose, mdiImagePlus } from "@mdi/js";
+import "@material/mwc-icon-button/mwc-icon-button";
+import "./ha-svg-icon";
+import "@polymer/paper-input/paper-input-container";
+import "@polymer/iron-input/iron-input";
+import { classMap } from "lit-html/directives/class-map";
```

Comment:
Do we have a import order style guide? Ever since @bramkragten made changes to the linter it seems like we don't keep absolute imports at the top and etc. Is there some thing I missed?

Reply:
We think we never had a rule for import order. I did a cleanup at some point I think...

Reply:
We had more of an agreement. Atleast when we first started it was absolute then { } imports and then component imports. Just looks clean to me. Down to understand the new rule. But I think import order can make a file look very bad if not delt with

Reply:
Sorry. I don't mean to be weird about this small thing just seems like we stopped actually doing anything about it

Reply:
All imports are done automatically 🤷‍♂️ so it is not unwillingness I guess but...